### ISSUE TITLE
Client stats on partitioned topics missing some aggregated values

### ISSUE THREAD

**Issue by sleungtoast** [apache/pulsar#4432]
**Describe the bug**
In `ProducerStatsRecorderImpl.updateCumulativeStats`, it aggregates
```
numMsgsSent
numBytesSent
numSendFailed
numAcksReceived
totalMsgsSent
totalBytesSent
totalSendFailed
totalAcksReceived
```
but doesn't aggregate
```
sendMsgsRate
sendBytesRate
latencyPctValues[]
```
leading to incomplete stats for a parent partitioned topic producer. The consumer stats have a similar problem.

**To Reproduce**
1. Create a partitioned topic
2. Create a producer on that topic
3. Call `producer.getStats()`, the rate and latency values will always be 0

**Expected behavior**
The "parent" producer of a partitioned topic should have fully populated aggregated stats from all its "children" producers

**Comment by hellozepp** [comment]
I'm interested in trying to solve this issue, tks

**Comment by hellozepp** [comment]
#4482

**Comment by codelipenghui** [comment]
@hellozepp Are you working for this issue? 
@sleungtoast Can we move this issue to next milestone(2.4.1 or 2.5.0)?

**Comment by hellozepp** [comment]
@codelipenghui Yes,There are some changes that need to be confirmed.

**Comment by sleungtoast** [comment]
@codelipenghui Yes we have a workaround so we are not blocked on this.

**Comment by codelipenghui** [comment]
move to 2.7.0

**Comment by tisonkun** [comment]
Closed as stale and no one showed interest any longer. Please open a new issue if it's still relevant to the maintained versions.
